L1Trigger/Phase2L1Taus/interface/HPSPFTauQualityCut.h

```diff
@@ -1,18 +1,18 @@
-#ifndef L1Trigger_Phase2L1Taus_L1HPSPFTauQualityCut_h
-#define L1Trigger_Phase2L1Taus_L1HPSPFTauQualityCut_h
+#ifndef L1Trigger_Phase2L1Taus_HPSPFTauQualityCut_h
+#define L1Trigger_Phase2L1Taus_HPSPFTauQualityCut_h
 
 #include "FWCore/ParameterSet/interface/ParameterSet.h"         // edm::ParameterSet
 #include "DataFormats/L1TParticleFlow/interface/PFCandidate.h"  // l1t::PFCandidate
 #include <string>                                               // std::string
 #include <vector>                                               // std::vector
 
-class L1HPSPFTauQualityCut {
+class HPSPFTauQualityCut {
```

Comment:
I think you should keep the original `L1HPSPFTauQualityCut` name, **or** move the class inside the `l1t` namespace

Reply:
Done